sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyCurveName.cs

```diff
@@ -20,4 +20,41 @@ public enum KeyCurveName : uint
 
         All = uint.MaxValue
     }
+
+    public static class KeyCurveNameExtensions
+    {
+        public static KeyCurveName ParseFromString(string value)
+        {
+            switch (value)
+            {
+                case "P-256":
+                    return KeyCurveName.P256;
+                case "P-256K":
+                    return KeyCurveName.P256K;
+                case "P-384":
+                    return KeyCurveName.P384;
+                case "P-521":
+                    return KeyCurveName.P521;
+                default:
+                    return KeyCurveName.Other;
+            }
+        }
+
+        public static string ParseToString(KeyCurveName curve)
```

Comment:
>public static string ParseToString(KeyCurveName curve) [](start = 8, length = 54)

The method name ParseToString seems a bit odd to me.

Reply:
`ToString()`?

Reply:
`AsString()`?

Reply:
I like AsString. I will change it to that